public/app/percona/pmm-dump/PMMDump.service.ts

```diff
@@ -38,6 +38,24 @@ export const PMMDumpService = {
   async delete(dumpIds: string[]) {
     await api.post<void, DeleteDump>(`${BASE_URL}/Delete`, { dump_ids: dumpIds });
   },
+  async dowload(dumpIds: string[]) {
+    for (const dumpId of dumpIds) {
+      api.get<AxiosResponse<Blob>, DeleteDump>(`/dump/${dumpId}.tar.gz`).then((response: AxiosResponse<Blob>) => {
+        const blob = new Blob([response.data], { type: 'application/gzip' });
+        const url = window.URL.createObjectURL(blob);
+
+        const a = document.createElement('a');
```

Comment:
We could probably reuse the a element for all the dumps.

Reply:
you're right, I'll try